source/clear-linux/concepts/security.rst

```diff
@@ -0,0 +1,207 @@
+
+.. _security:
+
+|CL| Security 
+*********************
+
+|CL|, as it is distributed and maintained upstream, aims to make 
+systemic and layered security conscious decisions, which are both
+performant and practical. This security philosophy is rooted within
+the project's codebase and operating culture.
+
+Below are just some examples of how:
+
+
+.. contents:: :local:
+   :depth: 2
+
+
+
+Security in Updates
+===================
+
+The |CL| team believes in the potential benefits of 
+software security through open sourcing, incremental updates, and 
+rapidly resolving known security advisories.
+ 
+
+
+The latest Linux codebase
+--------------------------------
+
+The |CL| Project operates around the idea of open sourcing 
```

Comment:
How about something like this?

Line 32: The |CL| Project relies on the open-source model to incorporate ... 
Line 33: ... the best new features and optimizations from the upstream Linux kernel.
Line 34: REMOVE